packages/body/README.md

```diff
@@ -95,133 +95,128 @@ Body component should only be used in the context of a page, to format long-form
 		</sup>
 	</p>
 
-	<main>
```

Comment:
Removing `main` tag as in GOLD, they have custom styles for element this which are not included in the body styles https://auds.service.gov.au/packages/body/tests/site/